File Name Adjustment

ivanbuha opened the pull request:
Adjusted File Name to Match Class Name 
class name - DefaultRabbitMQPersistentConnection
file name - DefaultRabbitMQPersisterConnection

mvelosop commented:
Hi @ivanbuha, thanks for the heads-up and the PR, however the file was updated and now has a merge conflict, if you solve this or create a new PR I'll merge it right away.

dnfclas commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet-architecture/eShopOnContainers?pullRequest=746) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-architecture/eShopOnContainers?pullRequest=746) before we can accept your contribution.<br/><br/>:x: ivanbuha [sign now](https://cla.dotnetfoundation.org/dotnet-architecture/eShopOnContainers?pullRequest=746)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-architecture/eShopOnContainers?pullRequest=746) it.</sub>

ivanbuha commented:
Hi @mvelosop , I updated pull request. In addition, I fixed file name for IRabbitMQPersistentConnection interface, which I might missed during initial pull request. Thank you.

mvelosop commented:
Thanks @ivanbuha, merging now!
